SharedContent/cs-winui-template/WinUI3TemplateCs/SampleConfiguration.cs

```diff
@@ -0,0 +1,38 @@
+﻿//*********************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF
+// ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY
+// IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR
+// PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.
+//
+//*********************************************************
+
+using System;
+using System.Collections.Generic;
+using Microsoft.UI.Xaml.Controls;
+
+namespace WinUI3TemplateCs
+{
+    public static class Settings
+    {
+        public const string FeatureName = "Windows App SDK Template C# Sample";
+        public static string CurrentTheme;
+    }
+
+    public partial class MainPage : Page
+    {
+        private readonly List<Scenario> scenarios = new()
```

Comment:
This pattern was inherited from the UWP samples repo, where the `MainPage.xaml.cs` file is shared among all samples to save disk space and make it easier to update all 200+ samples at one go (by updating the shared copy). Under that model, sample-specific stuff goes into `SampleConfiguration.cs`.

If we intend to follow the same pattern here, then the scenario list needs to live somewhere outside the shared copy (since it changes from sample to sample).

Even if we decide that each sample gets its own copy of `MainPage.xaml.cs`, we might want to keep it separate anyway, because that would make it easier to update all the samples at once (by replacing the MainPage.xaml.cs with an updated version).